ouroboros-consensus/test-consensus/Test/Consensus/BlockchainTime/Simple.hs

```diff
@@ -13,6 +13,7 @@
 module Test.Consensus.BlockchainTime.Simple (tests) where
 
 import           Control.Exception (SomeException, fromException)
+import qualified Control.Monad.Class.MonadTimer
```

Comment:
```suggestion
import qualified Control.Monad.Class.MonadTimer as Unused
```
Is this better?

Reply:
I see. Ok.

But move it down so it's together with
```haskell
import           Control.Monad.Class.MonadTime
import           Control.Monad.IOSim
```